### ISSUE TITLE
Permission overlay shows repetitive keywords

### ISSUE THREAD

**Issue by marians** [giantswarm/roadmap#2364]
Found in: https://happa.g8s.gorilla.eu-central-1.aws.gigantic.io/organizations/giantswarm-demos/access-control

1. Select the `read-all` role
2. Select the Permissions tab
3. Scroll down to the `pods/log` resources where `G` (get) and `L` (list) are highlighted
4. Hover the permissions column

A big tooltip appears with content `get, get, get, get, get, get, get, get, get, get, ... list, list, list, list, list, list, list, list, `.

<img width="1297" alt="image" src="https://user-images.githubusercontent.com/273727/232456800-2d8b9598-d6d4-408d-b606-c8db3a0e13b4.png">

Each permission keyword should appear only once. Here it should be `get, list`.

**Comment by kuosandys** [comment]
We could filter for unique verbs in `happa`, but looking at the `read-all` Cluster Role on several installations (including `gorilla`) show that the following permission rule is repeated multiple times in the resource definition, resulting in the long list we see in the UI:
```
- apiGroups:
  - ""
  resources:
  - pods/log
  verbs:
  - get
  - list
```
I think we need to move [defining and appending this policy rule](https://github.com/giantswarm/rbac-operator/blob/ac525e88213144a6cc99db86aab40912a31de729/service/controller/defaultnamespace/resource/clusterroles/create.go#L116) out of the `for` loop in `rbac-operator`.

**Comment by kuosandys** [comment]
Released with `rbac-operator` v0.33.3
